Resources/Prototypes/Entities/Buildings/airlock_types.yml

```diff
@@ -130,3 +130,110 @@
 
   - type: Icon
     sprite: Buildings/Doors/airlock_security_glass.rsi
+
+- type: entity
+  parent: Airlock
+  id: AirlockMaint
+  name: Maintenance Hatch
+  components:
+  - type: Sprite
+    sprite: Buildings/Doors/airlock_maint.rsi
+
```

Comment:
Do not separate components

Reply:
I copy-pasted the previous door prototypes. I'll fix this for the whole file.